guide/getting-started/getting-started.md

```diff
@@ -34,61 +34,61 @@ Illustration sources
 
 # Getting started
 
-Welcome to bitcoin design, you're in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating bitcoin products. You'll find a brief summary of each concept on this page, click through if you would like to dig deeper into a specific topic. Note that this is not meant as a comprehensive overview of all topics bitcoin, but only the ones we think are important to know as a designer working in this space.
+Welcome to Bitcoin Design Guide Guide! You’re in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating Bitcoin products. You’ll find a summary of concepts on this page, click through if you would like to dig deeper into a specific topic. The guide isn't a comprehensive overview of all Bitcoin topics, but only the ones you should know as a designer working in this space.
 
 ---
 
-**[Why bitcoin is unique]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})**
+**[Why Bitcoin is unique]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})**
 
 The internet has opened an infinite design space for information and communication. Bitcoin adds the new dimension of transferring value that could bring similar change as previous waves of innovation.
 
 ---
 
-**[Why design for bitcoin]({{ '/guide/getting-started/why-design-for-bitcoin/' | relative_url }})**
+**[Why design for Bitcoin]({{ '/guide/getting-started/why-design-for-bitcoin/' | relative_url }})**
 
 Bitcoin is fundamentally open and participatory and allows designers and creators to explore and shape experiences for users around the globe.
 
 ---
 
 **[Open design]({{ '/guide/getting-started/open-design/' | relative_url }})**
 
-Open design is about global collaboration that balances the free exchange of design information, the independence of individual designers and the collective power of collaboration.
+Open design is about global collaboration that balances the free exchange of design information, individual designers' independence, and the collective power of collaboration.
```

Comment:
I prefer the original phrasing, it's simpler to read.

Reply:
You're right, reverted in https://github.com/BitcoinDesign/Guide/commit/8796246fd2cb09bb9065848f6f28145b3566637d